weave-scope-cloud-service/README.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: guides
+shorttitle: Using Weave Scope Cloud Service to Visualize and Monitor Docker Containers
+title: Using Weave Scope Cloud Service to Visualize and Monitor Docker Containers
+description: Use Weave Scope to monitor and visualize docker containers.
+tags: weave scope, docker containers visibility, cluster, swarms
+permalink: /guides/weave-scope/weave-scope-cloud-service.html
+sidebarpath: /start/weave-scope-cloud
+sidebarweight: 51
+---
+
+Weave Scope automatically detects and monitors every host, container and process in your infrastructure, builds a map showing their inter-communications and then presents an up-to-date view of your infrastructure in a web interface. You can visualize, monitor and control your distributed applications and troubleshoot bottlenecks, memory leaks or any other issues. It does this without requiring changes to your code or configuration, and without having to make declarations about your infrastructure that become out-of-date and stale. 
+
+While Weave Scope works with Docker and the Weave network, neither is required. Weave Scope can be deployed to any infrastructure, and works well in all cloud and bare-metal environments.
```

Comment:
Whilst technically true, we distribute scope as a Docker image, so I wouldn't say it doesn't require Docker.